bevy/django-celery-beat
# [SGP-22292] Make django-celery-beat Django 3-compatible

This PR pulls in the 2.2.0 release of the upstream `django-celery-beat` package, which adds Django 3 compatibility.

This is actually already in use: it was intended to be deployed with the Django 3 upgrade, but was deployed sooner with the Celery 5 upgrade: https://github.com/startupgrind/Platform/pull/11177 

### Manual QA

1. Checkout the branch from https://github.com/startupgrind/Platform/pull/10810, which pins `django-celery-beat` to this commit.
2. Run `BEVY_VARIATION=demo docker-compose -f docker-compose-lts.yml up` (you may also need to add the `--build` flag). When the app starts, you should not see any errors relating to this package.
